pyinteraph/centrality_analysis.py

```diff
@@ -0,0 +1,604 @@
+                                     pdb = args.pdb)
+
+    # get graph nodes and edges
+    nodes = graph.nodes()
+    edges = graph.edges()
+    # print nodes
+    info = f"Graph loaded! {len(nodes)} nodes, {len(edges)} edges\n" \
+           f"Node list:\n{np.array(identifiers)}\n"
+    sys.stdout.write(info)
+
+    ############################ CENTRALITY ############################
+
+    # Function map of all implemented measures
+    function_map = {'hubs' : get_hubs,
+                    'degree': get_degree_cent, 
+                    'betweenness': get_betweeness_cent,
+                    'closeness': get_closeness_cent,
+                    #'communicability' : get_communicability_betweenness_cent,
+                    'group_betweenness' : get_group_betweenness_cent,
+                    'group_closeness' : get_group_closeness_cent,
+                    'edge_betweenness' : get_edge_betweenness_cent}
+    
+    # Get list of all centrality measures
+    if args.cent is not None:
+        # Get node list if present
+        if args.group is not None:
+            node_list = convert_input_to_list(args.group, identifiers)
+        else:
+            node_list = args.group
+
+        # Find group names based on user request
+        # Find all non group centralities
+        if "all" in args.cent:
+            centrality_names = all_cent
+        # Find all node centralities
+        elif "node" in args.cent:
+            centrality_names = node
+        # Find all group centralities if node list specified
+        elif "group" in args.cent and args.group is not None:
+            centrality_names = group
+        # Throw error if group is requested but node list is not specified
+        elif ("group" in args.cent or \
+            # One of the group centralities is requested
+            len([cent for cent in args.cent if cent in group]) > 0) \
+            and args.group is None:
+            error_str = "A group of residues must be specified to calculate " \
+                        "group centrality (see option -g). Exiting..."
+            log.error(error_str)
+            exit(1)
+        # Find all edge centralities
+        elif "edge" in args.cent:
+            centrality_names = edge
+        else:
+            # Get list of specified centrality names
+            centrality_names = args.cent
+
+        # Check sorting options
+        # sorting arg is either node or in centrality names
+        if not (args.sort_node == "node" or args.sort_node in centrality_names):
+            err_str = "The node sorting centrality argument (option -b) must be one " \
+                      "of the node centrality arguments used in option -c. Exiting..."
+            log.error(err_str)
+            exit(1)
+
+        # sorting arg is either edge or in centrality names
+        if not (args.sort_edge == "edge" or args.sort_edge in centrality_names):
+            err_str = "The edge sorting centrality argument (option -d) must be one " \
```

Comment:
You can repeat here in the error string what are the valid values (formatting the choices' list, not rewriting them one by one because it would be difficult to track in case they change).